examples/graphql/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "opentelemetry-plugin-graphql-example",
   "private": true,
-  "version": "0.15.0",
+  "version": "0.16.0",
```

Comment:
Why 0.16.0?
Besides that we should also update the version below in `dependencies` section.
The same is applicable to other examples.

Reply:
you're right. examples need other work anyway. i'll just revert it